src/components/application_manager/include/application_manager/policies/policy_handler.h

```diff
@@ -93,13 +93,10 @@ class PolicyHandler : public PolicyHandlerInterface,
   virtual void OnPermissionsUpdated(const std::string& policy_app_id,
                                     const Permissions& permissions) OVERRIDE;
 
-#ifdef EXTERNAL_PROPRIETARY_MODE
-  void OnSnapshotCreated(const BinaryMessage& pt_string,
-                         const std::vector<int>& retry_delay_seconds,
-                         int timeout_exchange) OVERRIDE;
-#else   // EXTERNAL_PROPRIETARY_MODE
+#if !(defined(PROPRIETARY_MODE) || defined(EXTERNAL_PROPRIETARY_MODE))
```

Comment:
@LuxoftAKutsan 
Description. Why it's not defined by interface?
Why HTTPExt? Looks like its not defined for external proprietary.